docker-compose.override.yaml

```diff
@@ -47,7 +47,7 @@ services:
       - "10254:10254"
       - "10255:10255"
     environment:
-      AZURE_COSMOS_EMULATOR_PARTITION_COUNT: 5 
+      AZURE_COSMOS_EMULATOR_PARTITION_COUNT: 6 
       AZURE_COSMOS_EMULATOR_ENABLE_DATA_PERSISTENCE: true
       AZURE_COSMOS_EMULATOR_IP_ADDRESS_OVERRIDE:
```

Comment:
@simonnineteen90 Had a question on the `AZURE_COSMOS_EMULATOR_IP_ADDRESS` variable. Why does it not have a value? Does it just need to be made available to Docker Compose? Was reading the [Cosmos docs](https://learn.microsoft.com/en-us/azure/cosmos-db/how-to-develop-emulator?tabs=docker-linux%2Cjavascript&pivots=api-nosql#start-the-emulator) and saw this is listed as an optional value so wasn't sure if that meant it still needed to be listed but could just be left blank?

Reply:
Good spot, left over from a copy paste. I think you can override the where cosmos is running on your machine but not sure why. I have removed it for now.